models/intel/bert-large-uncased-whole-word-masking-squad-0001/description/bert-large-uncased-whole-word-masking-squad-0001.md

```diff
@@ -0,0 +1,59 @@
+# bert-large-uncased-whole-word-masking-squad-0001
```

Comment:
Is it possible to shorten the model name? 
e.g.: `bert-luwwm-squad`, `bert-large-wwm-squad`

Reply:
@eizamaliev, it is original model name (bert-large-uncased-whole-word-masking), from transformers, I think shortcut can confuse users

Reply:
@eaidova and what? We have description of this model right below. Can you figure out something about model from it, can't you?

Reply:
not all users read description, it is a fact. Model naming with whole name is clear that shourtcut.
And it is intel model, not a public, no one intel model  does not have abbriviature (e.g. person-vehicle-bike-detection-security-1016, not named as pvb-detection-security, right?)

Anyway, the decision about model naming in @vshampor  hands :)

Reply:
I would like to keep the current naming please.

Reply:
@eizamaliev you know that we try to keep model names as close to original as possible.